rca/programmes/models.py

```diff
@@ -396,6 +396,7 @@ class ProgrammePage(TapMixin, ContactFieldsMixin, BasePage):
         choices=(
             ("1", "Applications closed. Please check back soon."),
             ("2", "Still accepting applications"),
+            ("3", "Applications closed"),
```

Comment:
```suggestion
            ("3", "Applications closed."),
```

According to the image - I think options 2 and 3 here need  updating with a `.` at the end. 

![image](https://github.com/user-attachments/assets/cf5a82c7-d978-45fb-b8d4-acf41cdb549d)